pkg/types/alibabacloud/machinepool.go

```diff
@@ -31,7 +31,7 @@ type MachinePool struct {
 
 	// SystemDiskCategory defines the category of the system disk.
 	//
-	// +kubebuilder:validation:Type=DiskCategory
+	// +kubebuilder:validation:Enum="";cloud_efficiency;cloud_essd
```

Comment:
This line should be removed entirely. The validation should be inherited from the validation on the `DiskCategory` type.

Reply:
Ok I have fixed this. Thanks!